runelite-client/src/main/java/net/runelite/client/plugins/itemcharges/ItemChargePlugin.java

```diff
@@ -483,6 +513,10 @@ public void onWidgetLoaded(WidgetLoaded widgetLoaded)
 							log.debug("Reset expeditious bracelet");
 							updateExpeditiousBraceletCharges(MAX_SLAYER_BRACELET_CHARGES);
 							break;
+						case ItemID.FALADOR_SHIELD:
+							log.debug("Reset falador shield");
+							updateFaladorShieldCharges(MAX_FALADOR_SHIELD_CHARGES);
```

Comment:
I don't think this makes any sense since destroying the shield doesn't reset the charges.